akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/javadsl/Replicator.scala

```diff
@@ -215,6 +218,11 @@ object Replicator {
    */
   final case class UpdateTimeout[A <: ReplicatedData](key: Key[A]) extends UpdateFailure[A]
 
+  /**
+   * The [[Update]] couldn't be performed because the entry has been deleted.
+   */
+  final case class UpdateDataDeleted[A <: ReplicatedData](key: Key[A]) extends UpdateResponse[A]
```

Comment:
> This still looks the same, forgot to push the change?

this is in Typed javadsl, and I hadn't update that. Changed to be consistent with the other, i.e. `extends UpdateResponse`